testbed/testbed/validator.go

```diff
@@ -75,437 +66,3 @@ func (v *PerfTestValidator) RecordResults(tc *TestCase) {
 		errorCause:        tc.errorCause,
 	})
 }
-
-// CorrectnessTestValidator implements TestCaseValidator for test suites using CorrectnessResults for summarizing results.
-type CorrectnessTestValidator struct {
```

Comment:
This extraction can be split because it is a trivial code move.

Reply:
Ok, I removed these changes from this PR.